src/main/resources/youtube/settings/host/values/strings.xml

```diff
@@ -591,9 +591,11 @@ Settings → Autoplay → Autoplay next video"</string>
     <string name="revanced_bypass_ambient_mode_restrictions_summary_on">Ambient mode is enabled in battery saver mode.</string>
     <string name="revanced_bypass_ambient_mode_restrictions_summary_off">Ambient mode is disabled in battery saver mode.</string>
     <string name="revanced_disable_ambient_mode_title">Disable ambient mode</string>
-    <string name="revanced_disable_ambient_mode_summary">Disables ambient mode.</string>
+    <string name="revanced_disable_ambient_mode_summary_on">Ambient mode is disabled.</string>
+    <string name="revanced_disable_ambient_mode_summary_off">Ambient mode is enabled.</string>
     <string name="revanced_disable_ambient_mode_in_fullscreen_title">Disable ambient mode in fullscreen</string>
-    <string name="revanced_disable_ambient_mode_in_fullscreen_summary">Disables ambient mode for fullscreen only.</string>
+    <string name="revanced_disable_ambient_mode_in_fullscreen_summary_on">Ambient mode is only disabled for fullscreen.</string>
```

Comment:
Isn't `Ambient mode is disabled/enabled in fullscreen` enough when toggle is on/off?

Reply:
Then there would be a case where the `Disable ambient mode` toggle is enabled and the `Disable ambient mode in fullscreen description` would still say "Ambient mode is enabled in fullscreen."

This implies that Ambient is currently working in fullscreen, but in reality it is disabled everywhere.

Ideally, there are two completely independent toggles:

1. `Disable ambient mode under player`
2. `Disable ambient mode in fullscreen`

Reply:
> This implies that Ambient is currently working in fullscreen, but in reality it is disabled everywhere.

Does it matter though, as the option gets grayed out, if `Disable Ambient mode` is toggled on. 

Also, there's other options which has similar description style. An example being the `Hide Shorts shelves`.